test/unit/driver/test_delegated.py

```diff
@@ -211,16 +211,40 @@ def test_ansible_connection_options_when_managed(mocker, _instance):
         'cloud-user',
         'ansible_private_key_file':
         '/foo/bar',
-        'connection':
-        'ssh',
+        'ansible_connection':
+        'smart',
         'ansible_ssh_common_args': ('-o UserKnownHostsFile=/dev/null '
                                     '-o ControlMaster=auto '
                                     '-o ControlPersist=60s '
                                     '-o IdentitiesOnly=yes '
                                     '-o StrictHostKeyChecking=no'),
     }
+
     assert x == _instance.ansible_connection_options('foo')
 
+    z = mocker.patch(
```

Comment:
Please use better var names communicating the intent more clearly.

Reply:
Hi, @webknjaz ok, got it.

Reply:
fixed

Reply:
@webknjaz Tests were passed please review.